src/components/ItemList/index.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState, useEffect } from 'react';
+import { updateItemDate } from '../Utils/firestore';
+import { differenceInHours } from 'date-fns';
+
+function ItemList({ itemName, docId, lastPurchasedDate }) {
+  const [isChecked, setIsChecked] = useState(false);
+  const [isDisabled, setIsDisabled] = useState(false);
+  const [token] = useState(() => window.localStorage.getItem('tcl18-token'));
+  const toggle = () => setIsChecked(!isChecked);
+
+  const editProduct = (id) => {
+    updateItemDate(token, id)
+      .then(() => {
+        console.log('product updated');
+      })
+      .catch((error) => {
+        console.log(error);
+      });
+  };
+
+  useEffect(() => {
+    if (isChecked) {
+      updateItemDate(token, docId).then(() =>
```

Comment:
I think It's not necessary two calls to function `updateItemDate`, because checked is not updated. But, It's possible I'm wrong.

Reply:
That part of code was removed.